app/views/comments/_comment.html.erb

```diff
@@ -48,21 +48,23 @@
                 <%= t("shared.collective") %>
               </span>
             <% end %>
-            <% if comment.user_id == comment.commentable.author_id %>
+            <% comment.commentable.is_a?(Poll) ? nil : if comment.user_id == comment.commentable.author_id %>
```

Comment:
Not entirely sure if a ternary operator is the way to go here. Comments welcome.